### PR TITLE
fetch Github credentials from the environment

### PR DESCRIPTION
**Pull request by arschles**
This patch makes the server slightly more in-line with 12-factor methodologies. Also, since the GitHub access token is one of the more sensitive configuration values, this is a good place to start

Closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ github-service-broker-ruby
 
 [![Build Status](https://travis-ci.org/cloudfoundry-samples/github-service-broker-ruby.png?branch=master)](https://travis-ci.org/cloudfoundry-samples/github-service-broker-ruby) (master)
 
+[![Docker Repository on Quay](https://quay.io/repository/deis/cloudfoundry-github-service-broker/status "Docker Repository on Quay")](https://quay.io/repository/deis/cloudfoundry-github-service-broker)
```
Comment (SocalNick) [inline README.md]: I'm not sure we want to point people at a particular Docker image repository.

File: service_broker/Makefile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+DOCKER_REPO ?= quay.io/
+DOCKER_USER ?= deis
+DOCKER_VERSION ?= latest
+
+DOCKER_IMAGE ?= ${DOCKER_REPO}${DOCKER_ORG}/cloudfoundry-github-service-broker:${VERSION}
```
Comment (SocalNick) [inline service_broker/Makefile]: Same as above, but also, DOCKER_ORG doesn't appear to be defined?

**Comment by SocalNick** [comment]
Please rebase your changes so the

File: service_broker/service_broker_app.rb [hunk 3]
```diff
@@ -119,7 +119,6 @@ def self.app_settings
   end
 
   def github_service
-    github_credentials = self.class.app_settings.fetch("github")
-    GithubServiceHelper.new(github_credentials.fetch("username"), github_credentials.fetch("access_token"))
```
Comment (simonleung8) [inline service_broker/service_broker_app.rb]: Taking out this piece of code is a breaking change for people who have not configured their ENV but are still using a yaml file. To achieve what you intend to do, we recommend use the ENV VAR if it exists, else go back default behavior which is reading from the yaml file.

**Comment by SocalNick** [comment]
Closing due to inactivity

**Pull request closed without merging**